plugins/opa/pkg/opa/opa_test.go

```diff
@@ -100,6 +100,27 @@ requestinsightsinfo[description] {
 }
 `
 
+const maxReplicas = `
+package fairwinds
+
+foo := {"s": "foo"}
+
+envMaxReplicasDeployments {
```

Comment:
please, also add a test using
```envMaxReplicasDeployments[actionItem] {
    ...
}```

which is usually how policies are defined

Reply:
done